app/client/src/widgets/InputWidgetV2/widget/index.tsx

```
@@ -640,8 +641,10 @@ class InputWidget extends BaseInputWidget<InputWidgetProps, WidgetState> {
   }
 
   static getDerivedPropertiesMap(): DerivedPropertiesMap {
+    const parsedDerivedProperties = parseDerivedProperties(derivedPropertyFns);
+
     return merge(super.getDerivedPropertiesMap(), {
```

Biome

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)